Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM ghcr.io/siderolabs/ca-certificates:v1.1.0 AS image-ca-certificates
 FROM ghcr.io/siderolabs/fhs:v1.1.0 AS image-fhs
 
 # runs markdownlint
-FROM node:18.7.0-alpine AS lint-markdown
+FROM docker.io/node:18.7.0-alpine3.16 AS lint-markdown
```

Comment:
we were using two node images:
- node:18.7.0-alpine
- node:18.7.0-alpine3.16
now moved to using the same, since both are the same.

```bash
❯ crane digest docker.io/node:18.7.0-alpine
sha256:02a5466bd5abde6cde29c16d83e2f5a10eec11c8dcefa667a2c9f88a7fa8b0b3

❯ crane digest docker.io/node:18.7.0-alpine3.16
sha256:02a5466bd5abde6cde29c16d83e2f5a10eec11c8dcefa667a2c9f88a7fa8b0b3
```